blog/_posts/2018-06-10-bsp.md

```diff
@@ -0,0 +1,214 @@
+IntelliJ 2018.2 nightly. To try the prototype out, follow the instructions in
+[this
+document](https://gist.github.com/jvican/0d738cc0aa47afe5dbbdfe6ea1bd8694).
+
+To learn more about the Build Server Protocol, check out the
+[slides](http://jorge.vican.me/slides/BSP.pdf) of my talk with Justin at
+Scalasphere 2018.
+
+## Next steps
+
+The next steps will focus on two main areas: adoption and user impact. First,
+in order to ease support of all build tools in IntelliJ, the Scala Center
+will keep collaborating with Jetbrains to improve IntelliJ's BSP import for
+Bloop and other build tools. The goal is to bring BSP import to a
+production-ready level where users can benefit from a build-tool-agnostic
+import project that is both faster and more reliable.
+
+Second, there is work to do to create a better synergy between build tool
+authors and IDE implementors. The [Scala Tooling Protocol Working Group]()
+that the Scala Center created a few months ago will help us discuss and
+refine the protocol so that all tooling developers in our community can
+implement the protocol at scale.
+
+Third, I'd like to collaborate more closely with developers working on Scala
+language servers like [Metals] or [Dotty IDE] to bring Scala support to all
+editors. There is a big chunk of Scala developers that don't use IntelliJ and
+I'd like to make sure that they can benefit from a smooth developer
+experience too.
+
+Last but not least, the Build Server Protocol is a language-agnostic protocol
+that can be modified to add support for any programming language. Thus, I'd
+like to reach out to folks in other programming language communities to find
+out ways we can pull our efforts together.
+
+## How to get involved
+
+The best way to share your thoughts on the Build Server Protocol or to get
+involved in its development is to open an issue or pull request to
+[`scalacenter/bsp`](https://github.com/scalacenter/bsp). Most of the features
+in the specification have tickets and review discussions elaborating on the
+design goals, like this [closed](https://github.com/scalacenter/bsp/issues/3)
+and [open](https://github.com/scalacenter/bsp/issues/14) tickets.
+
+If you want to help integrating BSP in your favorite editor or IDE, or you'd
+like to help test the integrations, join our
+[`scalacenter/bsp`](https://gitter.im/scalacenter/bsp/) Gitter channel and
+let's discuss about ways we can work together.
+
+## Acknoledgements
+
+I'd like to thank my colleague at the Scala Center, Ólafur Páll Geirsson, for
+being the co-author of the Build Server Protocol. His help and insights, as
+well as his close understanding of [LSP], have been fundamental in designing
+a protocol that puts the ergonomics of the clients (IDEs) first.
+
+Finally, I'd like to thank Justin Kaeser and, more broadly, JetBrains for
+being part of this initiative. It feels great to collaborate with other
+parties to have a positive impact in the tools we provide to the Scala
```

Comment:
s/in/on/
s/we provide/we are providing/ - I think that makes it sound more active!